githubservice/githubservice_test.go

```diff
@@ -106,5 +106,26 @@ func Test(t *testing.T) {
 
 			fmt.Println("Issue count: " + strconv.Itoa(len(issues)))
 		})
+
+		g.It("Should find commits for a repo", func() {
+			commits, err := s.loadCommitsForRepo("RobotsAndPencils", "marvin", "")
+
+			Expect(commits).ToNot(BeNil())
+			Expect(err).To(BeNil())
+		})
+
+		g.It("Should find PR commits for a repo", func() {
+			commits, err := s.loadCommitsFromAllRepoPRs("RobotsAndPencils", "marvin")
+
+			Expect(commits).ToNot(BeNil())
+			Expect(err).To(BeNil())
+		})
+
+		g.It("Should find commits to master", func() {
+			commits, err := s.CommitsToMaster("RobotsAndPencils", "marvin")
+
+			Expect(commits).ToNot(BeNil())
+			Expect(err).To(BeNil())
+		})
```

Comment:
It's pretty much consistent with what we have already, so that's good enough for me. I'll poke through the rest of the changes now too. Thanks.